docs-2.0/3.ngql-guide/12.vertex-statements/1.insert-vertex.md

```diff
@@ -19,6 +19,28 @@ prop_value_list:
   [prop_value [, prop_value] ...]
 ```
 
+<!---
+
+// Ignore existing index to insert vertex.
+```ngql
+INSERT VERTEX [IF NOT EXISTS] [IGNORE_EXISTED_INDEX] [tag_props, [tag_props] ...]
+VALUES VID: ([prop_value_list])
+tag_props:
+  tag_name ([prop_name_list])
+prop_name_list:
+   [prop_name [, prop_name] ...]
+prop_value_list:
+   [prop_value [, prop_value] ...]   
+```
+- `IGNORE_EXISTED_INDEX`：Indexes will not be updated after insert vertex. We recommend that using `IGNORE_EXISTED_INDEX` on the first import, which can greatly improve performance.
+
+  !!! danger
+
+        Existing indexes will not be deleted when using the `IGNORE_EXISTED_INDEX`, so incorrect data may be read during lookups. You need to rebuild the index, for more information, see [REBUILD INDEX](../3/../14.native-index-statements/4.rebuild-native-index.md。
```

Comment:
why do we need to rebuild the existing index?

Reply:
Lookup error data may occur if indexes are not rebuilt

Reply:
> 

by design